src/redis-client.ts

```diff
@@ -0,0 +1,8 @@
+import { createClient } from '@vercel/kv';
+
+const redisClient = createClient({
+  url: process.env.REDIS_URL!,
+  token: process.env.REDIS_TOKEN!,
```

Comment:
Instead of a non-null assertion, could we perform a check to see if REDIS_URL / REDIS_TOKEN exist, and handle the error condition in some way?

That way, we will be warned if the env vars are not set.